src/main/resources/static/scripts.js

```diff
@@ -9,16 +9,71 @@ function lightModeToggle() {
     }
 }
 
+function openModal() {
+    const modal = document.getElementById("myModal");
+
+    modal.style.display = "block";
+}
+
+function resetModal() {
+    const modalContent = document.getElementById("modal-content");
+    let closeSpan = document.createElement("span");
+    closeSpan.setAttribute("class", "close");
+    closeSpan.innerText = '×';
+
+    modalContent.innerHTML = '';
+    modalContent.appendChild(closeSpan);
+    modalWindow();
+}
```

Comment:
This function is needed so that each time the modal is used, it contains only the desired information and not a record of all the previous modal contents.